cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -103,18 +103,18 @@ preferred method of deploying it is to use the CDAP CLI.
 
 - Load the Artifact::
 
-    $ cdap-cli.sh load artifact target/WikipediaPipeline-3.3.0-SNAPSHOT.jar name WikipediaPipelineApp version
-    3.3.0-SNAPSHOT
+    $ cdap-cli.sh load artifact target/WikipediaPipeline-<version> name WikipediaPipelineApp version
+    <version>
     Successfully added artifact with name 'WikipediaPipelineApp'
 
 - Create an application using LDA as the clustering algorithm::
 
-    $ cdap-cli.sh create app wiki-lda WikipediaPipelineApp 3.3.0-SNAPSHOT user resources/wikipedia-lda.json
+    $ cdap-cli.sh create app wiki-lda WikipediaPipelineApp <version> user resources/wikipedia-lda.json
     Successfully created application
 
 - Create an application using K-Means as the clustering algorithm::
 
-    $ cdap-cli.sh create app wiki-kmeans WikipediaPipelineApp 3.3.0-SNAPSHOT user resources/wikipedia-kmeans.json
+    $ cdap-cli.sh create app wiki-kmeans WikipediaPipelineApp <version> user resources/wikipedia-kmeans.json

```

Comment:
what do you mean a version replacement? Like the rst file will be built, instead of static?